notification/src/main/java/com/iluwatar/notification/App.java

```diff
@@ -0,0 +1,30 @@
+package com.iluwatar.notification;
+
+/**
+* With the DAO pattern, we can use various method calls to
+* retrieve/add/delete/update data without directly interacting
+* with the data source. The below example demonstrates basic CRUD
+* operations: select, add, update, and delete.
+*/
+public final class App {
+
+  /**
+   * This is not called.
+   */
+  private App() {
+  }
+
+  /**
+   * Program entry point.
+   *
+   * @param args command line args
+   */
+  public static void main(final String[] args) {
+    final String courseId = "CSE427";
+    final String semester = "Fall21";
+    final String department = "Engineering";
+    final FormRegisterCourse form = new FormRegisterCourse(courseId, semester, department);
+    form.submit();
```

Comment:
It's possible to use local variable type inference (keyword `var`) here. Additionally, it would be nice to have some comments that describe what happens. This is an example code for others to read, after all.